tutorial.md

```diff
@@ -223,25 +198,47 @@ The index page of the documentation reproduces the content from the repository R
+```sh
+.venv\Scripts\activate
+```
 
-A Conda environment for the project can be created by running in a terminal the command
+You should see `(.venv)` on the left of your terminal window. Once you have activated the environment you can use the following command to install any packages within the `.venv` environment
 
 ```sh
-conda create -y -n {project_slug} -c conda-forge python
+uv pip install numpy
 ```
 
-This will create a new environment with name `{project_slug}` (which you should replace with the relevant project slug value for your project), installing the latest version of Python using the package hosted on the [community driven `conda-forge` channel](https://conda-forge.org/). To make this Conda environment the current active environment run
+The environment can be deactivated using
 
 ```sh
-conda activate {project_slug}
+deactivate
 ```
 
-again replacing `{project_slug}` with the relevant project slug for your package.
+You can even use an arbitrary Python version to create `uv` environments, and `uv` will fetch that Python version if it is not available locally
+
+```sh
+uv venv --python 3.11.6
+```
 
-An alternative to Conda is the [`venv` module](https://docs.python.org/3/library/venv.html) built-in to the Python standard library. This has the advantage of being available in any Python (3.3+) environment, but unlike Conda will not by itself allow you to use a different Python version from the system level install. In contrast to Conda which by default creates all environments in a shared user-level directory (if using Miniconda, by default in a directory `miniconda3/envs` in your user or home directory), the `venv` module requires being passed a path in which to create the directory containing the files associated with the virtual environment. A common pattern is to store the virtual environment files in a directory `.venv` within the root directory of the project repository. This can be achieved by running
+Alternatively, you can use the [`venv` module](https://docs.python.org/3/library/venv.html), which is slower, has less features, and is not developed rapidly when compared to `uv`, but is built-in to the Python standard library. Similar to `uv`, a virtual environment can be created using
```

Comment:
I'd say that the specific point made previously that the built in `venv` module does not allow you to manage the base Python version used in the environment would be useful to keep here. I would also say that 'developed rapidly' is not necessarily always a benefit 🙃 particularly for infrequent / less experienced users for which a more stable tool might be better.

Reply:
Should be back now